src/Symfony/Component/VarDumper/Caster/DOMCaster.php

```diff
@@ -23,7 +23,7 @@
 class DOMCaster
 {
     private const ERROR_CODES = [
-        \DOM_PHP_ERR => 'DOM_PHP_ERR',
+        0 => 'DOM_PHP_ERR',
```

Comment:
Do we need to keep this entry? The way I read [the RFC](https://wiki.php.net/rfc/deprecations_php_8_4#deprecate_dom_php_err_constant) it will no longer be needed:

> PR https://github.com/php/php-src/pull/11927 got rid of the last use of DOM_PHP_ERR in PHP. It is a non-standard and non-well-defined error code. The last use was for an out-of-memory situation but that's inconsistent as we normally use INVALID_STATE_ERR for that.

Reply:
To me, yes. It's targeting 5.4, and we better be careful and just remove deprecation with the minimum possible updates, and without changing any behavior if possible.

Reply:
Well, what I have in mind is adding this entry only in case the used PHP version is before 8.4 (or maybe when the constant is actually defined). Inlining the constant value makes it harder to identify them later when we want to clean up things.

Reply:
Even with the plain string next to it? It is stored in a constant array and this will require a bit of extra logic where the array constant is called. Still worth it to you?